lib/qldb.connect.js

```diff
@@ -71,8 +118,8 @@ class qldbConnect {
                     return false;
                 })
             });
-        } catch{
-            throw new Error('Create record failed');
+        } catch (err) {
+            throw new Error('Create record failed', err);
         } finally {
             await this.closeSession()
         }
```

Comment:
Hello @Steit , that finally block on create API causing issues on normal servers(instead of lambda). I think we might need to remove that finally block